shared-helpers/src/locales/es.json

```diff
@@ -1018,7 +1019,7 @@
   "t.neighborhood": "Comunidad",
   "t.next": "Siguiente",
   "t.no": "No",
-  "t.noMatchingListings": "No hay listados coincidentes",
+  "t.noMatchingListings": "No hay ningún listados coincidentes",
```

Comment:
This should say "listado coincidente" in singular, current grammar is wrong

Reply:
Fixed! thanks!